pytools/tag.py

```diff
@@ -162,9 +162,9 @@ class UniqueTag(Tag):
 # }}}
 
 
-TagsType = FrozenSet[Tag]
-TagOrIterableType = Union[Iterable[Tag], Tag, None]
-T_co = TypeVar("T_co", bound="Taggable")
+TagSet = FrozenSet[Tag]
+ToTagSetConvertible = Union[Iterable[Tag], Tag, None]
+Taggable_co = TypeVar("Taggable_co", bound="Taggable")
```

Comment:
Should this have a `TypeVar(..., covariant=True)`?

Reply:
Upon further thought, what these IMO really should be is the `Self` type from Py 3.11.

Reply:
(I've dropped the type variable, with deprecation.)

Reply:
I didn't look before how it's used, but yeah, that makes sense!